db/migrations/005_add_ward_id_to_scheduled_calls_table.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE scheduled_calls_table ADD ward_id int REFERENCES wards (id);
```

Comment:
A lot of the documentation has INTEGER over int. Do we use int elsewhere?

Reply:
int is simply an alias for integer (see https://www.postgresql.org/docs/current/datatype.html#DATATYPE-TABLE)

Reply:
coolio 👍